cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -2690,15 +2690,16 @@ macro(resolve_dependency_absl)
         stacktrace
         status
         statusor
-        strerror
         str_format_internal
+        strerror
         strings
         strings_internal
         symbolize
         synchronization
         throw_delegate
         time
-        time_zone)
+        time_zone
+        wyhash)
```

Comment:
Most of the cmake changes here seem to just be differences in sorting (due to my locale?), but the omission of `wyhash` seems to be legitimately new.